src/common/wincmn.cpp

```diff
@@ -2961,10 +2961,10 @@ void wxWindowBase::OnSysColourChanged(wxSysColourChangedEvent& WXUNUSED(event))
             event2.SetEventObject(win);
             win->GetEventHandler()->ProcessEvent(event2);
         }
-
         node = node->GetNext();
     }
 
+    event.Skip(true);
```

Comment:
Why do we have to do this now? Is there some default handling of this event at the underlying toolkit level? If so, this probably ought to be emphasized in the documentation (unless we prefer to always forward this event to the native handler).